teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super()
```

Comment:
- [ ] Good job Inheriting the `Teacher` class from `Person`. As I mentioned in the above comment, you should ensure that the `Constructor` extends parent's constructor by adding `@specialization `and a `parameter` for it i.e.:
```ruby
  ...
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    ...
  end
  ...
```